src/env/node.js

```diff
@@ -109,20 +111,29 @@ export default {
 	},
 	setup () {
 		process.env.NODE_ENV = "test";
+		interceptedConsole = interceptConsole();
 	},
 	done (result, options, event, root) {
 		makeCollapsible(root)
 		render(root, options);
 
 		if (root.stats.pending === 0) {
+			logUpdate.clear();
+
+			let {messages, originalConsole} = interceptedConsole;
+			restoreConsole(originalConsole);
+
+			// Replay all the suppressed messages from the tests
+			for (let message of messages) {
```

Comment:
I would probably encapsulate this in a function. Or encapsulate all this functionality in a class.